alibi/utils/tensorflow/wrappers.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
+
+from alibi.utils.wrappers import blackbox_wrapper
+from functools import wraps
+from typing import Callable
+
+
+@blackbox_wrapper(framework='tensorflow')
+def wrap_blackbox_predictor(func: Callable) -> Callable:
+    """
+    A decorator that converts the first argument to `func` to a `np.array` object and converts the output to a tensor.
+    """
+    @wraps(func)
+    def wrap(*args, **kwargs):
+        X, *others = args
+        if isinstance(X, tf.Tensor) or isinstance(X, tf.Variable):
+            X = X.numpy()
```

Comment:
What is the reason to convert back to numpy before calling the predict fn?

Reply:
Because the black-box prediction function is assumed to consume and output numpy arrays.